entrypoint.py

```diff
@@ -19,8 +23,10 @@
 INFO = '\033[34m'
 NOTICE = '\033[33m'
 
-if 'RELEASE_TOKEN' not in os.environ:
-    print(ERROR + "RELEASE_TOKEN needs to be set in env. Exiting." + ENDC)
+deploy_key = os.environ.get('DEPLOY_KEY')
+if not deploy_key and 'RELEASE_TOKEN' not in os.environ:
+    print(f"{ERROR}Either RELEASE_TOKEN or DEPLOY_KEY needs to be set in env. "
+          f"Exiting.{ENDC}")
```

Comment:
If you are going to change the prints to using f"" for errors, can you update them all in one go so there's a common pattern to them?

Reply:
:+1: A common pattern is worth having. I can also replace the f strings to string concatenation, do you have any preference? Personally, I have a slight tendency to f strings as they are a bit more compact, but that is all a question of preference and habit. I can also change everything to f strings.